### PR TITLE
Schema fixes

### PR DESCRIPTION
**Pull request by washort**
A few initial cleanups post-SQLAlchemy switch. Second commit is just a stylistic change in the APIs, let me know what you think.

### CODE REVIEW

**Comment by washort** [comment]
Updated to master and fixed style issues. This shouldn't change functionality, any chance you can review it soon?

**Comment by arikfr** [comment]
I really want to wrap a final release of v1.0 these days. I'm not going to introduce such a big change a moment before without proper review, even if it's not supposed to change functionality.

File: migrations/versions/3905cabd818e_json_fields.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+# revision identifiers, used by Alembic.
+revision = '3905cabd818e'
+down_revision = '4b86d00d91d7'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.alter_column('organizations', 'settings', type_=postgresql.JSONB, postgresql_using='settings::jsonb')
```
Comment (arikfr) [inline migrations/versions/3905cabd818e_json_fields.py]: JSONB was introduced in Postgres 9.4, while many Redash deployments are on Postgres 9.3.

Comment (washort) [inline migrations/versions/3905cabd818e_json_fields.py]: OK I'll stick with regular `json` field type.

**Comment by arikfr** [comment]
This is too "cold" to ever be merged. If you still think some of the changes here are relevant, please open an issue to discuss first. Thanks.

Closing this now.

**Pull request closed without merging**